docs/tasks/step-2.md

```diff
@@ -1,45 +1,67 @@
+
+Finally there is a style section where you can define the styles for your component using CSS. For example background color, text color, object size, font etc.
+
+All three technologies work together to create dynamic web pages and apps.
+
+![Development cake visual](./devel-cake.png)
 
-Open `/src/components/BookCard.vue` and have a look at it, it is a simple vue component with a template, script and style section. The template is where you structure your component using html. The script is where the logic goes using javascript, inside the script you can also define `props` (properties) which allow dynamic variables to be passed into the component. Finally there is a style section where you define the styles for that component using css.
+## b) The BookCard component is currently not being used by the homepage. 
+Follow the steps below to add it:
+1. Open `src/pages/index.vue` (the homepage component).
+2. Find the `<div />` that has step 2 underneath it.
+3. Add the component `<book-card />` underneath the 'Step 2 ...' line. Your code should look something like this:
 
-The BookCard component is currently not being used by the homepage. Open `src/pages/index.vue` (the homepage component) to add it.
+```
+<div class="cards-container">
+  <!-- Step 2: Add the <book-card/> component here. -->
+  <book-card />
+</div>
+```
 
-To add the component use a tag that has the components name in kebab case. In this case we want to render the `BookCard.vue` component so we add `<book-card ... />` to the template of `index.vue`.
+In the script section we have pre-filled a few lines of code to help show this component on your screens. Save your changes and have a look at your webpage.
 
-In the script for `index.vue` we already return a list of the `favoriteBooks` for the component to use in the template. This is done using the store getter `getFavoriteBooks` which gets all of the books from `books.json` that have `favorite: true` set. We can use `v-for` to loop through each book in the favorite books list and create a book card component for each of the favorite books. The `v-for` requires a unique key for each component, we can pass the `isbn` to it as that is a unique identifier.
+You should see the following:<br/>
+![Home page with book-card component](./step-2-a.png)
 
-Finally we can pass data to the components props, `:prop-name="value"`, for example we can pass the title to the component by doing `:title="books.title"`.
+### c) Adding properties to the component
+For `index.vue` we already return a list of the `favoriteBooks` for the component to use in the template. Now we need to loop through the list of books to check which are favorites, using v-for (a built-in vue loop function). To do this successfully we need a unique identifier or key, e.g. isbn. We also want to display the title of the book. Add the isbn and title properties to your `<book-card />` component as follows:
```

Comment:
Might need to make this sound better. Change the last line to:

`Add the v-for loop, isbn and title properties to your `<book-card />` component as follows:`